### PR TITLE
chore: print a warning message about deprecated of customize uploader

### PR DESCRIPTION
**Pull request by chick-p**
## Why

This is a part of #530

## What

- print a warning message if user specifies `--domain` or `KINTONE_DOMAIN`.
- In interactive CLI, ask base-url instead of domain

## How to test

- execute `bin/cli.js dest/customize-manifest.json` with `--base-url` option or `KINTONE_DOMAIN`
- execute `bin/cli.js dest/customize-manifest.json` without setting of kintone environment 

## Checklist

- [x] Read [CONTRIBUTING.md](https://github.com/kintone/js-sdk/blob/master/CONTRIBUTING.md)
- [x] Updated documentation if it is required.
- [ ] Added tests if it is required.
- [x] Passed `yarn lint` and `yarn test` on the root directory.

### CODE REVIEW

File: packages/customize-uploader/README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ If you want to upload the customize files automatically when a file is updated,
     $ kintone-customize-uploader <manifestFile>
   Options
     --base-url Base-url of your kintone
-    --domain Domain of your kintone (If you set --base-url, this value is not necessary.)
+    --domain Domain of your kintone (This value is deprecated. Please you use --base-url.)
```
Comment (zaki-yama) [inline packages/customize-uploader/README.md:64]:
```suggestion
    --domain Domain of your kintone (This value is deprecated. Please use --base-url.)
```

File: packages/customize-uploader/README.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ If you want to upload the customize files automatically when a file is updated,
 
     You can set the values through environment variables
     base-url: KINTONE_BASE_URL
-    domain: KINTONE_DOMAIN (If you set `base-url`, this value is not necessary.)
+    domain: KINTONE_DOMAIN (This value is deprecated. Please you use KINTONE_BASE_URL.)
```
Comment (zaki-yama) [inline packages/customize-uploader/README.md:85]:
```suggestion
    domain: KINTONE_DOMAIN (This value is deprecated. Please use KINTONE_BASE_URL.)
```

File: packages/customize-uploader/bin/cli.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ const cli = meow(
     $ kintone-customize-uploader <manifestFile>
   Options
     --base-url Base-url of your kintone
-    --domain Domain of your kintone (If you set --base-url, this value is not necessary.)
+    --domain Domain of your kintone (This value is deprecated. Please you use base-url.)
```
Comment (zaki-yama) [inline packages/customize-uploader/bin/cli.js:28]:
```suggestion
    --domain Domain of your kintone (This value is deprecated. Please use --base-url.)
```

"you" seems unnecessary, and if we follow `--oauth-token`'s description, I prefer appending `--` prefix 👍

File: packages/customize-uploader/bin/cli.js [hunk 4]
```diff
@@ -46,7 +46,7 @@ const cli = meow(
 
     You can set the values through environment variables
     base-url: KINTONE_BASE_URL
-    domain: KINTONE_DOMAIN (If you set base-url, this value is not necessary.)
+    domain: KINTONE_DOMAIN (This value is deprecated. Please you use KINTONE_BASE_URL.)
```
Comment (zaki-yama) [inline packages/customize-uploader/bin/cli.js:49]:
```suggestion
    domain: KINTONE_DOMAIN (This value is deprecated. Please use KINTONE_BASE_URL.)
```

**Review by zaki-yama — COMMENTED** [review]
I tested that:

- When I pass `--domain` or `KINTONE_DOMAIN`, it displays a warning message
- When I pass nor `--domain/KINTONE_DOMAIN` or `--base-url/KINTONE_BASE_URL`, it asks `--base-url` instead of `--domain`

I suggested a few about the documentation. Please check 👍

**Comment by chick-p** [comment]
@zaki-yama 
Thank you for your review, and sorry my poor English...
I've fixed these!

**Review by zaki-yama — APPROVED** [review]
👍

**Review by nakajmg — COMMENTED** [review]
When i set both `KINTONE_DOMAIN` and `KINTONE_BASE_URL`, it displays a waring message.

Is it expected behavior ?

**Comment by chick-p** [comment]
@nakajmg 
Thank you for your comment and nice catch-up!

I guess it is expected behavior.
Because I want users to remove `KINTONE_DOMAIN` cause the unused environment value confuses.

cc @zaki-yama

**Review by nakajmg — APPROVED** [review]
👍

**Pull request merged**